File: CRUD/Python/Sistema con MySQL/Productos.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
# Written By Ismael Heredia in the year 2017

class Producto(object):

   def __init__(self, id="", nombre="", descripcion="",precio="",id_proveedor="",fecha_registro=""):
       self.__id = id
       self.__nombre = nombre
       self.__descripcion = descripcion
       self.__precio = precio
       self.__id_proveedor = id_proveedor
       self.__fecha_registro = fecha_registro
              
   def getId(self):
       return self.__id

   def setId(self, id = ""):
       self.__id = id

   def getNombre(self):
       return self.__nombre

   def setNombre(self, nombre = ""):
       self.__nombre = nombre

   def getDescripcion(self):
       return self.__descripcion

   def setDescripcion(self, descripcion = ""):
       self.__descripcion = descripcion
    
   def getPrecio(self):
       return self.__precio

   def setPrecio(self, precio = ""):
       self.__precio = precio
       
   def getId_proveedor(self):
       return self.__id_proveedor

   def setId_proveedor(self, id_proveedor = ""):
       self.__id_proveedor = id_proveedor
       
   def getFecha_registro(self):
       return self.__fecha_registro

   def setFecha_registro(self, fecha_registro = ""):
       self.__fecha_registro = fecha_registro
       
   def toString(self):
       return "Producto{id=%d, nombre=%s, descripcion=%s, precio=%d, fecha_registro=%s, id_proveedor=%d}" % (self.__id,self.__nombre,self.__descripcion,self.__precio,self.__fecha_registro,self.__id_proveedor)

   def destroy(self):
       self.__id = None
       self.__nombre = None
       self.__descripcion = None
       self.__precio = None
       self.__id_proveedor = None
       self.__fecha_registro = None

File: Ejercicios POO/Javascript/POO/productos.js
// Written By Ismael Heredia in the year 2016

function Producto(id_producto,nombre_producto,descripcion,precio,id_proveedor,fecha_registro){
	this.id_producto = id_producto;
	this.nombre_producto = nombre_producto;
	this.descripcion = descripcion;
	this.precio = precio;
	this.id_proveedor = id_proveedor;
	this.fecha_registro = fecha_registro;
}

Producto.prototype.getId_producto=function(){
	return this.id_producto;
};
Producto.prototype.setId_producto=function(id_producto){
	this.id_producto = id_producto;
};

Producto.prototype.getNombre_producto=function(){
	return this.nombre_producto;
};
Producto.prototype.setNombre_producto=function(nombre_producto){
	this.nombre_producto = nombre_producto;
};

Producto.prototype.getDescripcion=function(){
	return this.descripcion;
};
Producto.prototype.setDescripcion=function(descripcion){
	this.descripcion = descripcion;
};

Producto.prototype.getPrecio=function(){
	return this.precio;
};
Producto.prototype.setPrecio=function(precio){
	this.precio = precio;
};

Producto.prototype.getId_proveedor=function(){
	return this.id_proveedor;
};
Producto.prototype.setId_proveedor=function(id_proveedor){
	this.id_proveedor = id_proveedor;
};

Producto.prototype.getFecha_registro=function(){
	return this.fecha_registro;
};
Producto.prototype.setFecha_registro=function(fecha_registro){
	this.fecha_registro = fecha_registro;
};

Producto.prototype.toString=function(){
	return "[+] ID Producto : "+this.id_producto+"\n"+"[+] Nombre producto : "+this.nombre_producto+"\n"+
			"[+] Descripcion : "+this.descripcion+"\n"+"[+] Precio : "+this.precio+"\n"+
			"[+] Id Proveedor : "+this.id_proveedor+"\n"+"[+] Fecha Registro : "+this.fecha_registro;
};